### PR TITLE
En server: run performance test every 3 hours

### PR DESCRIPTION
**Pull request by chaodaiG**
Run performance test more frequently so that we can get a better sense of moving average changes as in: https://mako.dev/benchmark?benchmark_key=4790091207671808

/cc @sethvargo @MushuEE

### CODE REVIEW

File: prow/prowjobs/google/exposure-notifications-server/en-server.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ presubmits:
         secret:
           secretName: gcs-access-service-account
 periodics:
-  - cron: "0 8 * * *"  # Run at 00:00 PST
+  - cron: "* */3 * * *"  # Run every 3 hours
```
Comment (MushuEE) [inline prow/prowjobs/google/exposure-notifications-server/en-server.yaml:38]: Should this be `0 */3 * * *`? I don't know cron super well but I am not sure if the * in minutes would mean every minute every 3rd hour

Comment (chaodaiG) [reply prow/prowjobs/google/exposure-notifications-server/en-server.yaml]: You are right. It turns out that I'm even less familiar with cron than you are :) 
Fixed

**Pull request merged**